tests/test_generate_return_series.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import pandas as pd
+from quanteda.generate_return_series import generate_return_series
+
+def test_negative_annual_volatility():
+    """
+    Test case for ValueError when `annual_volatility` is negative.
+    """
+    with pytest.raises(ValueError) as error:
+        generate_return_series(
+            expected_annual_return=0.05,
+            annual_volatility=-0.2,
+            n_rows=3,
+            freq='D',
+            num_series=3,
+            dist='normal',
+            random_state=524,
+            start_date='2024-01-01'
+        )
+    assert str(error.value) == "annual_volatility < 0."
+
+def test_invalid_freq():
+    """
+    Test case for ValueError when an invalid `freq` is provided.
+    """
+    with pytest.raises(ValueError) as error:
+        generate_return_series(
+            expected_annual_return=0.05,
+            annual_volatility=0.2,
+            n_rows=3,
+            freq='ABC',
+            num_series=3,
+            dist='normal',
+            random_state=524,
+            start_date='2024-01-01'
+        )
+    assert str(error.value) == "Invalid frequency. Use 'D' for daily, 'H' for hourly, 'min' for minutely."
```

Comment:
Same issue as above here. Would this work?
```suggestion
    assert str(error.value) == "Invalid frequency. Use 'D' for day, 'H' for hour, 'min' for minute"
```

Reply:
thanks Merete